### ISSUE TITLE
in-memory database

### ISSUE THREAD

**Issue by newlix** [tekartik/sqflite#63]
Is in-memory database supported? I expected openDatabase(null) would return an in-memory db, but I got the following exception.
```
(
	0   CoreFoundation                      0x0000000103d891e6 __exceptionPreprocess + 294
	1   libobjc.A.dylib                     0x000000010341e031 objc_exception_throw + 48
	2   CoreFoundation                      0x0000000103e0a784 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x0000000103d0b898 ___forwarding___ + 1432
	4   CoreFoundation                      0x0000000103d0b278 _CF_forwarding_prep_0 + 120
	5   Runner                              0x0000000101477794 -[FMDatabase sqlitePath] + 84
	6   Runner                              0x0000000101477963 -[FMDatabase openWithFlags:vfs:] + 99
	7   Runner                              0x0000000101484a85 -[FMDatabaseQueue initWithPath:flags:vfs:] + 277
	8   Runner                              0x0000000101484c77 -[FMDatabaseQ<…>
```

**Comment by alextekartik** [comment]
Thanks for the report.
I added support for in-memory database in the new published version 0.9.0. null was not working on Android and crashing on iOS. It turns out `openDatabase(":memory:")` seems to work on both platform with the new delivery. Let me know

**Comment by newlix** [comment]
Thanks. It works on my ios simulator.
